src/it/it.iml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
```

Comment:
from CI:
> `Resource must start with 'Input' or 'Expected': com/puppycrawl/tools/checkstyle/checks/imports/imports.iml ==> expected: <true> but was: <false>`

please remove this file, it is not allowed to have files other than java files.

Reply:
@RhnSaxena, please do not resolve conversation. Author will do this, it is reminder to recheck all points for author.